src/main/java/org/graylog/integrations/dataadapters/GreyNoiseQuickIPDataAdapter.java

```diff
@@ -48,22 +49,22 @@
 import java.util.Map;
 import java.util.Objects;
 
-public class GreyNoiseDataAdapter extends LookupDataAdapter {
-    private static final Logger LOG = LoggerFactory.getLogger(GreyNoiseDataAdapter.class);
-    public static final String NAME = "GreyNoise";
```

Comment:
To maintain backwards compatibility, the adapter name constant cannot be renamed, since it is saved in the Mogno `db.lut_data_adapters` collection. You can find this with the Mongo CLI in the `config.type` field:
```
$ mongo
> use graylog
> db.lut_data_adapters.find()
``` 

```
{
        "_id" : ObjectId("617ff5032fcd864f475fc7f3"),
        "title" : "Test",
        "description" : "test",
        "name" : "test",
        "custom_error_ttl_enabled" : false,
        "config" : {
                "type" : "GreyNoise",
                "api_token" : {
                        "encrypted_value" : "53454353253454325",
                        "salt" : "342342"
                }
        }
}
```

Changing it would result in an error for adapters that exist already, so I think it needs to stay the same.
![image](https://user-images.githubusercontent.com/3423655/139701143-1b6afda5-1986-47e5-98c2-007145f4240f.png)

The class name changes should be OK, I think. Do you agree @waab76 @bernd?

Reply:
Reverted back to previous NAME for backwards compatibility

Reply:
Thanks. I agree. I think renaming them makes sense.